openmc/data/multipole.py

```diff
@@ -2,6 +2,7 @@
 
 import h5py
 import numpy as np
+from math import exp, erf, pi
```

Comment:
Per PEP8, standard-library imports should come first, so this should be moved up with `from numbers ...`. Also, I forgot to mention that `sqrt` should also come from `math`. Can you change that too?